src/demo_c.cpp

```diff
@@ -0,0 +1,15 @@
+#include "utils/unit_conversions.hpp"
+#include <cstdio>
+
+using workshop5::utils::unit_conversions::speedOfLightToMPS;
+
+int main();
+
+int main()
+{
+  double speed_in_c = 3.4;
+  double speed_in_mps = speedOfLightToMPS(speed_in_c);
```

Comment:
@Thanakrit-Anutrakulchai Fooled you! "MPS" is supposed to be "Mps"

Reply:
Ahh, I was wondering what I missed. Nice~